Lib/test/pythoninfo.py

```diff
@@ -452,6 +470,8 @@ def collect_info(info):
         collect_ssl,
         collect_sys,
         collect_sysconfig,
+        collect_test_socket,
+        collect_testsupport,
```

Comment:
I suggest to put test support at the end, since it can have the most side effects, and rename the function to collect_test_support.